clusterloader2/testing/density/suite.yaml

```diff
@@ -0,0 +1,8 @@
+- identifier: vanilla
+  configPath: testing/density/config.yaml
+  overridePaths: []
+
+- identifier: pod-pod-anti-affinity
```

Comment:
lets rename to pod-anti-affinity (drop the first pod), here and in the directory names as well.

Reply:
my thinking behind this was that we'll be adding pod-to-node affinity too (similar naming as the integration tests)

but since pod will be a common prefix, I think we can drop it :+1:

Reply:
I think we can use pod-anti-affinity/pod-affinity and node-anti-affinity/node-affinity.